src/lib/runtime_config/runtime_config.ml

```diff
@@ -11,279 +11,392 @@ let yojson_strip_fields ~keep_fields = function
 let opt_fallthrough ~default x2 =
   Option.value_map ~default x2 ~f:(fun x -> Some x)
 
+let result_opt ~f x =
+  match x with
+  | Some x ->
+      Result.map ~f:Option.some (f x)
+  | None ->
+      Result.return None
+
+module Json_layout = struct
+  module Accounts = struct
+    module Single = struct
+      type t =
+        { pk: (string option[@default None])
+        ; sk: (string option[@default None])
+        ; balance: Currency.Balance.Stable.Latest.t
```

Comment:
should this be `Currency.Balance.t` ?

Reply:
Indeed it should. Replaced all the `.Stable.Latest.t`s in the `Json_layout` types with `.t`s